06_building_stuff/src/components/Main.jsx

```diff
@@ -87,7 +93,7 @@ export default function Main() {
       </div>
       <h2>{h2}</h2>
       <div className="cards-container flex-center">
-        {restaurantData.map((item) => (
+        {filteredList.map((item) => (
```

Comment:
using `filteredList` instead of `restaurantData` because need to show the filtered data.